utils/rpc.ts

```diff
@@ -0,0 +1,79 @@
+import {ethers} from "ethers"
+
+export default class RPC {
+    constructor(public provider: ethers.providers.JsonRpcProvider) {}
+
+    sendAsync(method: string, arg: any[]) {
```

Comment:
can we explicitly mark it as 'async' and also specify a return type? (I guess by using a generic since it is used in functions that return `Promise<void>` or `Promise<someType>`